src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/BookingContact.kt

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity
+
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.IdClass
+import jakarta.persistence.Table
+import org.springframework.data.annotation.Immutable
+import java.io.Serializable
+
+@Entity
+@Immutable
+@Table(name = "v_booking_contacts")
+@IdClass(UniquePropertyId::class)
+data class BookingContact(
+  val videoBookingId: Long,
+
+  @Id
+  val contactType: String,
```

Comment:
Could this be an enum?  Then it would be clear what are the possible/supported values.

Reply:
Yes - I'll do this now.